packages/sdk/src/cosmwasmclient.ts

```diff
@@ -129,31 +172,47 @@ export class CosmWasmClient {
       );
     }
     return {
-      accountNumber: account.account_number,
+      accountNumber: account.accountNumber,
       sequence: account.sequence,
     };
   }
 
-  public async getAccount(address: string): Promise<CosmosSdkAccount | undefined> {
+  public async getAccount(address: string): Promise<Account | undefined> {
     const account = await this.restClient.authAccounts(address);
     const value = account.result.value;
-    return value.address === "" ? undefined : value;
+    return value.address === ""
+      ? undefined
+      : {
+          address: value.address,
+          balance: value.coins,
+          pubkey: value.public_key || undefined,
```

Comment:
Nice cleanup. I would consider converting the public_key from bech32 into an object (as mentioned above). I am pretty sure the parse code exists